### PR TITLE
geodig 1.0 (new formula)

### PR DESCRIPTION
**Pull request by nl5887**
geodig is a command line util that will resolve ip adresses and domain names to locations using the free maxmind geo database.

### CODE REVIEW

File: Library/Formula/geodig.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  homepage "https://github.com/dutchcoders/geodig"
+  url "https://github.com/dutchcoders/geodig/archive/v1.0.0.tar.gz"
+  sha1 "488be926d26f67c858f08ed324e26e9bb64b17d3"
+  head "https://github.com/dutchcoders/geodig.git"
+
+  bottle do
+    sha1 "fb10fc8d9b76f5d922d19ecb3c8cc0378ffdb4af" => :mavericks
+    sha1 "7fa6f1f652c74f5047bcccf260a336ad6f9ff6f0" => :mountain_lion
+    sha1 "2b7346b6b6991188de60eb1e35459357bad24e3f" => :lion
+  end
```
Comment (mikemcquaid) [inline Library/Formula/geodig.rb]: Please remove the bottle block; we'll add it when we pull.

File: Library/Formula/geodig.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+require "formula"
```
Comment (mikemcquaid) [inline Library/Formula/geodig.rb]: Please fix all `brew audit --strict geodig` failures.

File: Library/Formula/geodig.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    mkdir_p "src/github.com/dutchcoders/"
+    ln_s buildpath, "src/github.com/dutchcoders/geodig"
+
+    # Install Go dependencies
+    system "go", "get", "github.com/oschwald/maxminddb-golang"
+
+    # Build
+    system "go", "build", "-o", "geodig"
+
+    # Install binary and configuration
```
Comment (mikemcquaid) [inline Library/Formula/geodig.rb]: Please remove all the comments above.

File: Library/Formula/geodig.rb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    bin.install "geodig"
+  end
+
+  def post_install
+  end
```
Comment (mikemcquaid) [inline Library/Formula/geodig.rb]: Please remove this function.

File: Library/Formula/geodig.rb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+end
```
Comment (mikemcquaid) [inline Library/Formula/geodig.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by mikemcquaid** [comment]
This project appears to be a self-submission of a relatively new and irregularly developed project so it may not be a good fit for Homebrew. Perhaps you could maintain your own tap for it? I'll leave it open for now so other @Homebrew/owners can chip in. Thanks!

**Comment by nl5887** [comment]
Yes, the project is relatively new. I checked if there where any tools doing something the same, but not seen it. I personally think that it could be a good addition, but we'll hear from @Homebrew/owners what they think.

**Comment by DomT4** [comment]
Closing here. Please feel free to resubmit once your repository has 50+ Github Stars/Forks/PRs/etc - That helps show us there's significant interest in adding it to the core.

Thanks for your submission, we appreciate it!

**Pull request closed without merging**